wgengine/userspace.go

```diff
@@ -460,7 +460,7 @@ func (e *userspaceEngine) RequestStatus() {
 func (e *userspaceEngine) Close() {
 	e.Reconfig(&wgcfg.Config{}, nil)
 	e.linkMon.Close()
-	e.router.Close()
```

Comment:
Why don't we close the Router anymore?

Reply:
Sorry that must've been the fallout from some testing/refactoring I was doing.  The current ordering of router.Close -> device.Close should allow openbsd to down the interface before removing it, but I was seeing some double close issues related to setting the routes to an empty CIDR, which caused an error, and the error would force a secondary close.  There might still be an open bug on the openbsd side when this goes through, but I can't verify it at the moment.